packages/runtime/src/core.ts

```diff
@@ -118,6 +138,14 @@ export class FederationHost {
       origin: FederationHost;
     }>('beforeLoadShare'),
     loadShare: new AsyncHook<[FederationHost, string, ShareInfos]>(),
+    resolveShare: new SyncWaterfallHook<{
```

Comment:
Maybe we should tweak the internal devtool logic

Reply:
yes, chrom devtool can use this hook